BackofficeBundle/Resources/public/coffee/page/NodeFormView.coffee

```diff
@@ -4,7 +4,6 @@ NodeFormView = OrchestraModalView.extend(
     if @options.submitted
       displayRoute = appRouter.generateUrl "showNode",
               nodeId: $('#oo_node_nodeId', @$el).val()
-      Backbone.history.loadUrl(displayRoute)
```

Comment:
yes. displayMenu redirect if first parameter set. That's why i remove this line. The error was on the DOUBLON redirect